source/main.cpp

```diff
@@ -783,12 +475,13 @@ void action_about()
 {
     consoleClear();
 
-    printf(CONSOLE_RED "CIAngel by cearp and Drakia\n" CONSOLE_RESET);
+    printf(CONSOLE_RED "CIAngel by cearp and Drakia\n");
+    printf( CONSOLE_RESET);
```

Comment:
Not a fan of this particular change, the old code showed the intention better (Change to red, output text, change back), whereas this one someone may see the CONSOLE_RESET on its own and delete it by accident